libs/ledger-live-common/src/apps/support.ts

```diff
@@ -33,7 +33,7 @@ const appVersionsRequired = {
   Algorand: ">= 1.2.9",
   Polkadot: ">= 15.9300.0",
   Elrond: ">= 1.0.11",
-  Ethereum: ">= 1.10.1",
+  Ethereum: ">= 1.10.1-0",
```

Comment:
Unrelated right? Better remove it from this PR.

Reply:
It's on me, it's a quick fix to add this to fix the issue raised by Thierry ;)

Reply:
Ok change intended from @lambertkevin to mitigate the "FW update required" issue on LLM, that is triggered with this release because we had to force a new eth app version.